.gitignore

```diff
@@ -3,14 +3,10 @@
 *.diff
 
 # autogenerated during the build
-src/passes/WasmIntrinsics.cpp
+/src/passes/WasmIntrinsics.cpp
```

Comment:
what is the effect of adding the leading `/` everywhere?

Reply:
IIUC is just makes it a more precise match.

e.g.
```
*.o
```
vs
```
/*.o
```
The latter only match objects at the top level